src/Fantomas/CodePrinter.fs

```diff
@@ -963,8 +963,10 @@ and genExpr astContext synExpr =
 
         let genCompExprStatement astContext ces =
             match ces with
-            | LetStatement(isRecursive, binding) ->
-                let prefix = if isRecursive then "let rec " else "let "
+            | LetOrUseStatement(isRecursive, isUse, binding) ->
+                let prefix =
+                    sprintf "%s%s" (if isUse then "use " else "let ") (if isRecursive then "rec" else "")
```

Comment:
There's a slight change here, the trailing whitespace has been dropped. I think this would be flagged if the test had a recursive use / let case